src/makeRobotAccountant.js

```diff
@@ -6,7 +6,28 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+
+  const addNum = x => {
```

Comment:
Consider renaming the 'addNum' function to 'getSum' for better readability and alignment with the task description.